### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#852] [Galaxy Trivia] [answered]
> 🧬 isn't about 🔎 yourself. 🧬 is about ...

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment, marked as answer]
**Life isn't about finding yourself. Life is about creating yourself.**

This quote from George Bernard Shaw suggests that life is not solely about discovering a predetermined identity or purpose, but rather about actively shaping and defining who you are.

It emphasizes the idea that we have the power to create our own identities, values, and paths in life through conscious choices and actions. Instead of passively searching for a pre-existing self, this perspective encourages individuals to take an active role in self-discovery and personal growth.

By recognizing that life offers countless opportunities for growth, learning, and self-expression, we can actively engage in the process of shaping our identities and finding meaning. This involves exploring our passions, pursuing personal and professional goals, cultivating positive relationships, and living in alignment with our values.

In this context, "creating yourself" means taking ownership of your life, making intentional choices, and continuously evolving and adapting to new circumstances. It suggests that we have the agency to shape our own narratives and construct a meaningful existence based on our aspirations and values.

Ultimately, the phrase emphasizes the notion that life is not a fixed journey with a predetermined destination, but rather an ongoing process of self-creation and self-expression. It encourages individuals to embrace the opportunity to actively shape their lives, find personal fulfilment, and create a unique and authentic existence.
